servers/lib/server/handlers/invitetoteambycsvAnalyze.coffee

```diff
@@ -0,0 +1,101 @@
+os = require 'os'
+fs = require 'fs'
+hat = require 'hat'
+path = require 'path'
+Busboy = require 'busboy'
+parser = require 'csv-parse'
+{ generateFakeClient } = require './../client'
+
+module.exports.parserOpts = csvParseOpts =
```

Comment:
do not duplicate the configuration, use already exported parserOpts from invitetoteambycsv.coffee